package.json

```diff
@@ -24,13 +24,9 @@
   ],
   "devDependencies": {
     "bindings": "~1.2.1",
-    "commander": "^2.8.1",
-    "glob": "^5.0.14",
-    "request": "=2.81.0",
-    "node-gyp": "~3.6.2",
-    "readable-stream": "^2.1.4",
-    "tap": "~0.7.1",
-    "xtend": "~4.0.0"
+    "node-gyp": "^3.7.0",
+    "readable-stream": "^2.3.6",
+    "tap": "^0.7.1"
```

Comment:
did you intentionally change this to `^0.7.1` or did you mean to leave it unchanged at `~0.7.1`?

Reply:
Not intentionally, sorry. 
I mean to leave it unchanged at ~0.7.1 (before we find that root cause about CI result)
I'll change it back as soon.